### ISSUE TITLE
[A11y] Recommendation - Ensure That Region Page Has Footer Content

### ISSUE THREAD

**Issue by bsaadat** [PublicHealthEngland/coronavirus-dashboard#96]
On reviewing the site using VoiceOver iOS, it was noted that the Regional page does not have footer content. This means pages such as Accessibility cannot be reached from that page. We would recommend that the footer content is present on all pages.
 
**Steps to Recreate:**
1.    Navigate to URL
 
**This has been observed on the following pages:**
https://coronavirus.data.gov.uk/region
 
**Environment:**
iOS 13, Safari
 
**This issue is known to occur on:**
Believed to be global
 
**Reproduction rate:**
100% - Consistently reproducible. Occurs every time that it is attempted.
 
**Supporting material:**
Screenshot/Video
 
**Version:**
https://coronavirus.data.gov.uk/region

**Comment by Ha-san-ali** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [[A11y] Recommendation - Ensure That Region Page Has Footer Content](https://trello.com/c/wBv1cYkR/224-a11y-recommendation-ensure-that-region-page-has-footer-content)
